### ISSUE TITLE
Notifications API: inline replies

### ISSUE THREAD

**Issue by annevk** [mozilla/standards-positions#91]
Explainer: https://github.com/anitawoodruff/inline-notification-replies

Standard PR: https://github.com/whatwg/notifications/pull/132
HTML diff of that: https://whatpr.org/notifications/132/e9407eb...e308162.html

Since we don't have anyone actively maintaining the Notifications API, I figured I'd escalate this here.

I suspect we're either `defer` (minus myself, since I have helped with review) or `worth prototyping`.

**Comment by martinthomson** [comment]
This looks like an OK feature, but I think that it might be a little too low level a question for this repo.  It's a tiny feature.  Not sure that it makes sense to ask questions here at this level of detail.

Separately, I'm not sure that the proposed process for backward compatibility (treat these as simple buttons) works; some process for feature detection would seem to be necessary.

**Comment by annevk** [comment]
@martinthomson yeah, so what is the place for minor features? I'd rather not make the call alone on behalf of Mozilla as that feels somewhat conflict-of-interesty if I also help maintain the standard and for a number of things we don't have a clearly identified owner. (And even when there's a clearly identified owner it can be hard to get them to comment on the relevant GitHub issue.)

**Comment by ekr** [comment]
I'm less concerned with minor versus major than whether it fits into some broader set of APIs we are going to want; if so, it would be better to figure out the overall architecture

**Comment by dbaron** [comment]
Who would be likely to implement this if we were going to do so?  Maybe they would have feedback.

That said, it seems like a reasonable feature to me, so I'm fine with just marking as `worth prototyping`.

**Comment by dbaron** [comment]
That said, I'm also not entirely sure how to list this in the overall positions table, given that it's a feature within a spec...

**Comment by zcorpan** [comment]
The spec PR hasn't had updates since 2022. Closing with `defer` until there's renewed interest.
